packages/react/src/BaseStyles.module.css

```diff
@@ -37,17 +37,17 @@ table {
   }
 }
 
-[role='button']:focus:not(:focus-visible):not(.focus-visible),
-[role='tabpanel'][tabindex='0']:focus:not(:focus-visible):not(.focus-visible),
-button:focus:not(:focus-visible):not(.focus-visible),
-summary:focus:not(:focus-visible):not(.focus-visible),
-a:focus:not(:focus-visible):not(.focus-visible) {
+[role='button']:focus:not(:focus-visible):not(:global(.focus-visible)),
+[role='tabpanel'][tabindex='0']:focus:not(:focus-visible):not(:global(.focus-visible)),
+button:focus:not(:focus-visible):not(:global(.focus-visible)),
+summary:focus:not(:focus-visible):not(:global(.focus-visible)),
+a:focus:not(:focus-visible):not(:global(.focus-visible)) {
   outline: none;
   box-shadow: none;
 }
 
-[tabindex='0']:focus:not(:focus-visible):not(.focus-visible),
-details-dialog:focus:not(:focus-visible):not(.focus-visible) {
+[tabindex='0']:focus:not(:focus-visible):not(:global(.focus-visible)),
+details-dialog:focus:not(:focus-visible):not(:global(.focus-visible)) {
```

Comment:
@jonrohan I noticed that these did not have the `:global` wrapper, which I assume are needed for it to behave as expected.

Reply:
This is going in on the next release, anything specific that needs to be validated UI-wise before landing this? @jonrohan @hussam-i-am

Reply:
Not sure how to recreate this selector in the dom. Maybe drop a focus-visible class on a details-dialog element

Reply:
Double checked, doesn't look like it breaks anything. We have some duplicate styles for this in primer/css that's already being imported